matlab/schemas/+vis/FancyBar.m

```diff
@@ -0,0 +1,22 @@
+ %{
+vis.FancyBar (manual) # Moving Bar stimulus that keeps the size and speed
+# of the bar constant relative to the mouse�s perspective. 
```

Comment:
the comment must fit on one line (above) to be saved in the database.  These comments are not saved.